docs/source/FlatBuffers.md

```diff
@@ -4,8 +4,8 @@ FlatBuffers    {#flatbuffers_index}
 # Overview {#flatbuffers_overview}
 
 [FlatBuffers](@ref flatbuffers_overview) is an efficient cross platform
-serialization library for C++, C#, Go, Java, JavaScript, PHP, and Python
-(C and Ruby in progress). It was originally created at Google for game
+serialization library for C++, C#, C, Go, Java, JavaScript, PHP, and Python
+(Ruby and Swift in progress). It was originally created at Google for game
```

Comment:
Let's actually remove both these in progress languages, I don't like making promises.